### PR TITLE
feat: Add KB Brinkman dataset

### PR DESCRIPTION
**Pull request by EnnoMeijers**
feat: added Brinkman term source

### CODE REVIEW

**Comment by sdevalk** [comment]
@EnnoMeijers A test failed. If you update file `test/catalog.test.ts` by changing `expect(catalog.datasets.length).toBe(8);` to `expect(catalog.datasets.length).toBe(9);` it should work.

**Comment by EnnoMeijers** [comment]
> @EnnoMeijers A test failed. If you update file `test/catalog.test.ts` by changing `expect(catalog.datasets.length).toBe(8);` to `expect(catalog.datasets.length).toBe(9);` it should work.

Thanks, I'll give it a try!

**Comment by EnnoMeijers** [comment]
@sdevalk to bad, still problems with the test, another ideas?

**Comment by sdevalk** [comment]
@EnnoMeijers Ah, I see - the test probably fails because the `@id` of the Brinkman distribution is not unique; the NTA has the same `@id`: `http://data.bibliotheken.nl/thesp/sparql` (file `catalog/datasets/brinkman.jsonld`).

Is it possible to use a unique identifier, e.g. by using another IRI or by adding a distinguishing hash tag to the current IRI (such as `http://data.bibliotheken.nl/thesp/sparql#brinkman`)?

**Comment by ddeboer** [comment]
@sdevalk That’s right: every distribution’s URI needs to be unique; otherwise the [catalog query](https://github.com/netwerk-digitaal-erfgoed/network-of-terms-catalog/blob/master/src/catalog.ts#L35) mixes up distributions and their SPARQL queries.

File: catalog/datasets/brinkman.jsonld [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.bibliotheken.nl/id/dataset/brinkman",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Brinkman trefwoordenthesaurus "
```
Comment (ddeboer) [inline catalog/datasets/brinkman.jsonld:8]: Nitpick: the trailing space can go.

**Pull request merged**